### ISSUE TITLE
[Wizard] Include .xtext file in jar created by gradle

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#284]
[Wizard] Include .xtext file in jar created by gradle

to be able to reference one grammar from another one we need to package the grammar

**Comment by cdietrich** [comment]
current generation

```
sourceSets.all {
    resources.exclude '**/*.g', '**/*.xtext', '**/*.mwe2', '**/*.xtend', '**/*._trace'
}
```

**Comment by cdietrich** [comment]
same problem with non plain layout

**Comment by cdietrich** [comment]
fixed in 2.11.1
